inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/ManagerFetcher.java

```diff
@@ -300,6 +307,24 @@ private void dealWithFetchResult(TaskResult taskResult) {
         }
     }
 
+    private boolean triggerIsRunning(TriggerProfile newProfile) {
+        int type = ManagerOpEnum.getOpType(newProfile.getInt(JOB_OP)).getType();
+        if (ManagerOpEnum.ACTIVE.getType() != type || ManagerOpEnum.ADD.getType() != type) {
+            return false;
+        }
+        JobProfileDb jobProfileDb = agentManager.getJobProfileDb();
+        List<JobProfile> jobsByState = jobProfileDb.getJobsByState(StateSearchKey.ACCEPTED);
+        jobsByState.addAll(jobProfileDb.getJobsByState(StateSearchKey.RUNNING));
+        AtomicBoolean jobIsRunning = new AtomicBoolean(false);
+        jobsByState.forEach(jobProfile -> {
+            if (jobProfile.get(JOB_ID).equals(newProfile.get(JOB_ID))) {
```

Comment:
Will `jobProfile.get(JOB_ID)` be null?
Wouldn't it be better to use `Objects.equals()` here?

Reply:
done